apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -413,51 +456,101 @@ const getAppBySlug = async (appSlug: string) => {
   return app;
 };
 
-const getEventTypes = async (userId: number, teamId?: number) => {
```

Comment:
grouping all the event-types of a team/user -- so that its easier to show them in groups, Previously, we could only install the app for a single user on a single team, but now organizations may have many teams. Separating events by team will make it more manageable.
![Screenshot 2024-06-24 at 3 25 54 PM](https://github.com/calcom/cal.com/assets/32706411/a4694bac-bdf7-404c-88dc-2f061fa74eb2)